Create item-selector.html (#68)
Kentico/kontent-custom-element-samples: Item-selector/README.md

# Item selector

This custom element offers items from another project to be linked with your item

![screenshot](https://amend.cz/item-selector.png)

## Configuration

```json
{
    "projectid": "302946ce-a441-00e5-3dba-ec6ccc479168",
    "filter": "system.type=article"
}
```

> You need to specify the `projectid` parameter in order to make the element work. The optional `filter` parameter is for filtering just subitems of your project and you can use any filtering described in our documentation (separated by &):

Reviewer: ```suggestion
You need to specify the `projectid` parameter in order to make the element work. The optional `filter` parameter is for filtering just subitems of your project and you can use any filtering described in our [documentation](https://developer.kenticocloud.com/reference#content-filtering) except [`depth`](https://developer.kenticocloud.com/reference#section-linked-content-depth) and [`elements`](https://developer.kenticocloud.com/reference#projection) (separated by &):
```

Reviewer: I would also note what language is retrieved from the project and I would link the documentation describing the language fallbacks:
* https://docs.kenticocloud.com/tutorials/develop-apps/get-content/getting-localized-content#a-ignoring-language-fallbacks

Author: done, thanks!

https://developer.kenticocloud.com/reference#content-filtering

You can test this element by configuring https://kentico.github.io/custom-element-samples/Item-selector/item-selector.html url for your custom element.

## Output

The JSON output of such an element would look like this:

```json
"item_selector": {
        "type": "custom",
        "name": "item selector",
        "value": "[

>                    {\"codename\":\"article_1\",\"name\":\"Article 1\",\"type\":\"article\"},

Reviewer: I would consider adding language codename and/or guid in the selected response.

Author: done!

# Item selector

This custom element offers items from another project to be linked with your item

![screenshot](https://amend.cz/item-selector.png)

## Configuration

```json
{
    "projectid": "302946ce-a441-00e5-3dba-ec6ccc479168",
    "filter": "system.type=article"
}
```

You need to specify the `projectid` parameter in order to make the element work. The optional `filter` parameter is for filtering just subitems of your project and you can use any filtering described in our documentation (separated by &):

> https://developer.kenticocloud.com/reference#content-filtering

Reviewer: ```suggestion
[Content filtering documentation](https://developer.kenticocloud.com/reference#content-filtering)
```

> Within the filter you can also specify what language of your items you want to retrieve (a default language is returned ootb) - https://docs.kenticocloud.com/tutorials/develop-apps/get-content/getting-localized-content#a-ignoring-language-fallbacks

Reviewer: ```suggestion
Within the filter you can also specify what language of your items you want to retrieve (a default language is returned OOTB) - [https://docs.kontent.ai/tutorials/develop-apps/get-content/getting-localized-content#a-ignoring-language-fallbacks](https://docs.kontent.ai/tutorials/develop-apps/get-content/getting-localized-content#a-ignoring-language-fallbacks)
```

> You can test this element by configuring https://kentico.github.io/custom-element-samples/Item-selector/item-selector.html url for your custom element.

Reviewer: ```suggestion
You can test this element by configuring https://kentico.github.io/kontent-custom-element-samples/Item-selector/item-selector.html url for your custom element.
```
